motioneye/static/js/main.js

```diff
@@ -29,7 +29,8 @@ var pageContainer = null;
 var overlayVisible = false;
 var layoutColumns = 1;
 var fitFramesVertically = false;
-var layoutRows = 1;
+// Was removed 5 years ago, needs cleanup, a25dec205b70fbf0f75f8c2f424d4920a875f399
+// var layoutRows = 1;
```

Comment:
Let's clean this up already:
```suggestion
```

Reply:
The thing here is that the variable is not really used because it's function was removed but there are still some functions around where the variable is set:
```
$('#layoutRowsSlider').change(function () {
        layoutRows = parseInt(this.value);
        updateLayout();
        savePrefs();
    });
```
and

```
function applyPrefs(dict) {
    setLayoutColumns(dict['layout_columns']);
    fitFramesVertically = dict['fit_frames_vertically'];
    layoutRows = dict['layout_rows'];
    framerateFactor = dict['framerate_factor'];
    resolutionFactor = dict['resolution_factor'];

    if (fitFramesVertically) {
        getPageContainer().addClass('fit-frames-vertically');
    }
    else {
        getPageContainer().removeClass('fit-frames-vertically');
    }
}
``` 
I think those are some leftovers from the original functionality. It looks like there are a couple of things around from this one. Wouldn't it make sense to remove all those things that are not longer used anymore instead of deleting only this variable? I can do that tomorrow if you'd like?

Reply:
Ah right, I forgot about that. There is even the GUI toggle to set this, which should be removed all together. But we can do that in a dedicated PR.